packages/core/src/awsexplorer/activationShared.ts

```diff
@@ -25,8 +24,7 @@ export function registerToolView(viewNode: ToolView, context: vscode.ExtensionCo
     toolView.onDidExpandElement(e => {
         if (e.element.resource instanceof CdkRootNode) {
             // Legacy CDK metric, remove this when we add something generic
-            const recordExpandCdkOnce = once(() => telemetry.cdk_appExpanded.emit())
```

Comment:
Future idea: I wonder if we could use a [weakmap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap) or similar so that once() would be easier to use.